app/src/main/java/com/duycuong/weather/data/source/local/sqlite/LocationLocalDataSource.java

```diff
@@ -0,0 +1,107 @@
+package com.duycuong.weather.data.source.local.sqlite;
+
+import android.content.ContentValues;
+import android.content.Context;
+import android.database.Cursor;
+import android.database.sqlite.SQLiteDatabase;
+import android.database.sqlite.SQLiteOpenHelper;
+
+import com.duycuong.weather.data.model.Locationn;
+import com.duycuong.weather.data.source.LocationDataSource;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by DuyCương on 11/02/2018.
+ */
+
+public class LocationLocalDataSource extends SQLiteOpenHelper implements LocationDataSource {
+    public static int VERSION = 1;

```

Comment:
WARNING: [SuspiciousConstantFieldName] The field name indicates a constant but its modifiers do not
https://pmd.github.io/pmd-5.6.1/pmd-java/rules/java/naming.html#SuspiciousConstantFieldName